### DISCUSSION TITLE
Reconnect to GitHub - local files/folders deleted

### DISCUSSION THREAD

**Discussion by AndrewSRea** [community/community#37580] [Other Feature Feedback, Questions, & Ideas] [answered]
I recently sent my Macbook in for repair due to a repeating keystroke. Unfortunately, I did not backup my files and folders. Now upon receiving my Macbook back, all of the files/folders which were previously in my Visual Studio Code are now gone. Basically, I need to reconnect my (now new!) VSC to my GitHub account and my repos. I know how to clone my repos but how can I reconnect my VSC to my GitHub account so I can push future files/folders to my GitHub repos?

**Comment by DenverCoder1** [comment, marked as answer]
VSC should prompt you for a GitHub log in the first time you do an action involving GitHub. After that, it will be connected.

This article may help - https://code.visualstudio.com/docs/sourcecontrol/github#_authenticating-with-an-existing-repository

**Reply by AndrewSRea** [reply]
Thank you very much, DenverCoder1!
